app/graphql/mutations/sign_up.rb

```diff
@@ -6,6 +6,8 @@ class SignUp < BaseMutation
     argument :first_name, String, required: false
     argument :last_name, String, required: false
 
+    argument :avatar, String, required: false
```

Comment:
```suggestion
    argument :avatar, ImageUploaderType, required: false
```

```ruby
module Types
  class ImageUploaderType < Types::BaseInputObject
    argument :id, String
    argument :storage, String
    argument :metadata, ImageUploderMetadataType
  end
end

module Types
  class ImageUploderMetadataType < Types::BaseInputObject
    argument :size, String
    argument :filename, String
    argument :mime_type, String
  end
end
```

Maybe we can use default values for some of these arguments. Or we can calculate size ourself.